frontend/src/lit-components/app-root.js

```diff
@@ -23,6 +23,7 @@ export class AppRoot extends LitElement {
 
     switch (route) {
       case 'home': return html`
+        <nav-bar></nav-bar>
```

Comment:
I think there is a better way than adding the nav here. This is just temp to show what it looks like

Reply:
Cleanest way to attach the nav would probably be to have a page variable at the top of this app-root render function, setting it in the switch statement, then returning the html after the switch:
```
let page;

switch (route) {
    case 'home': page = '<home-page></home-page>'; break;
}

return html`
    <nav-bar></nav-bar>
    ${page}
`
```